c2corg_api/tests/views/test_outing.py

```diff
@@ -362,6 +362,112 @@ def test_post_missing_title(self):
         errors = body.get('errors')
         self.assertEqual(len(errors), 3)
 
+    def test_post_date_start_is_tomorrow(self):
+        tomorrow = (datetime.date.today() + datetime.timedelta(days=1)).strftime('%Y-%m-%d')
```

Comment:
With the UTC trick, adding 1 day can be seen as UTC today if test are launched after UTC noon.

```suggestion
        tomorrow = (datetime.date.today() + datetime.timedelta(days=2)).strftime('%Y-%m-%d')
```